static/src/stylesheets/module/commercial/glabs/_hosted.scss

```diff
@@ -642,3 +644,30 @@ $hosted-stripe: 28px;
 .hosted__popup-text {
     font-weight: 200;
 }
+
+.hosted__central-play-button {
+    @include f-headlineSans;
```

Comment:
Perhaps we could a generic class that can center elements in IE9+ whatever their dimensions and borders? You can do this using CSS `transform` properties:

```css
.translate-center {
  position: absolute;
  left: 50%;
  top: 50%;
  transform:translate(-50%, -50%)
}
```

You could put this in any non-statically positioned context and bam! It's centered. This would mean that
 - wouldn't need magic numbers like `18px` and `71px`, and
 - could re-use the class next time you wanted to center something